src/openrct2-ui/windows/Multiplayer.cpp

```diff
@@ -64,7 +64,7 @@ enum WINDOW_MULTIPLAYER_WIDGET_IDX {
 
 #define MAIN_MULTIPLAYER_WIDGETS \
     { WWT_FRAME,            0,  0,      339,    0,      239,    0xFFFFFFFF,                 STR_NONE },                 /* panel / background   */  \
-    { WWT_CAPTION,          0,  1,      338,    1,      14,     STR_MULTIPLAYER,            STR_WINDOW_TITLE_TIP },     /* title bar            */  \
+    { WWT_CAPTION,          0,  1,      338,    1,      14,     0xFFFFFFFF,                 STR_WINDOW_TITLE_TIP },     /* title bar            */  \
```

Comment:
Use `STR_NONE` here instead of a hardcoded magic number. If you could change the line above as well that would be nice.